### PR TITLE
CSI operator docs updates

### PR DESCRIPTION
**Pull request by samihan-dell**
# Description
- Fixing document review comments

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/583 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [X] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/operator/_index.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ The Dell CSI Operator is a Kubernetes Operator, which can be used to install and
 ## Prerequisites
 
 #### Volume Snapshot CRD's
-The Kubernetes Volume Snapshot CRDs can be obtained and installed from the external-snapshotter project on Github. Manifests are available here:[v6.2.1](https://github.com/kubernetes-csi/external-snapshotter/tree/v6.2.1/client/config/crd)
+The Kubernetes Volume Snapshot CRDs can be obtained and installed from the external-snapshotter project on Github. Manifests are available here: [v6.2.x](https://github.com/kubernetes-csi/external-snapshotter/tree/v6.2.1/client/config/crd)
```
Comment (rensyct) [inline content/docs/csidriver/installation/operator/_index.md:14]: Why is this changed to 6.2.x?

Comment (samihan-dell) [reply content/docs/csidriver/installation/operator/_index.md]: I reverted it. changed it to 6.2.1

File: content/docs/csidriver/installation/operator/_index.md [hunk 2]
```diff
@@ -250,18 +250,24 @@ Please refer to the _Troubleshooting_ section [here](../../troubleshooting/opera
...
     # Replace driver-namespace with the namespace where the Unity XT driver is installed
     $ kubectl get csiunity -n <driver-namespace>
     ```
-    use the object name of kind CSIUnity in `kubectl edit` command.
-
+    use the object name in `kubectl edit` command.
+    ```
+    $ kubectl edit <driver-object>/<object-name> -n <driver-namespace>
+    ```
+    For example - If the object name is of of kind CSIUnity.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/operator/_index.md:265]: Delete "of of kind"

File: content/docs/csidriver/installation/operator/_index.md [hunk 3]
```diff
@@ -250,18 +250,24 @@ Please refer to the _Troubleshooting_ section [here](../../troubleshooting/opera
...
     ```
     # Replace object-name with the object name of kind CSIUnity
     $ kubectl edit csiunity/<object-name> -n <driver-namespace>
     ```
-    and modify the installation. The usual fields to edit are the version of drivers and sidecars and the env variables.
+    and modify the installation. The usual fields to edit are the version of drivers and sidecars and the environment variables.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/operator/_index.md:270]: Add a comma after "drivers" and delete "and".

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. Two edits are required.

**Review by rensyct — APPROVED** [review]

File: content/docs/csidriver/installation/operator/_index.md (hunk 2 above)
Comment (samihan-dell) [reply content/docs/csidriver/installation/operator/_index.md]: Removed that text

**Review by rajkumar-palani — APPROVED** [review]
lgtm
